### DISCUSSION TITLE
Options to customise latency readouts

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#15796] [Ideas] [answered]
The utility of having the input latency and other such technical information readily available is really nice, but it also majorly obscures the difficulty graph and map time remaining.

![osu_2021-11-25_20-00-37](https://user-images.githubusercontent.com/47021866/143411273-da9a3628-c42d-456f-a965-8b9a7bcbd396.jpg)

some potential fixes could be adding an option to scale the readout, maybe one to move it to a different part of the hud, or what i personally think would be best is an option to choose which readouts you want showing (for instance hiding all but the input readout), currently, they are all on or all off, but having all four on obscures UI elements and is a bit annoying, so to keep the utility of the latency readouts in a more streamlined fashion, checkboxes for which readouts are wanted showing would be a good way to do this, this is obviously low priority, but it would be a nice feature

**Comment by smoogipoo** [comment, marked as answer]
These do not show “latency”. These should not be used to estimate/determine “latency”.

These are debugging displays, nothing more. They’re not intended to be shown 100% of the time unless you’re debugging something very specific.
